powershell/resources/psruntime/BuildTime/PsExtensions.cs

```diff
@@ -13,6 +13,13 @@ namespace Microsoft.Rest.ClientRuntime.PowerShell
 {
     internal static class PsExtensions
     {
+        public static PSObject AddMultipleTypeNameIntoPSObject(object obj, string multipleTag = "#Multiple")
```

Comment:
If you want to implement an extension method, the syntax should be like public static PSObject `AddMultipleTypeNameIntoPSObject(this object obj, string multipleTag = "#Multiple")`.  Please see [here ](https://learn.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/extension-methods)for details about extension method in C#.